insonmnia/hub/server.go

```diff
@@ -874,6 +874,11 @@ func (h *Hub) ProposeDeal(ctx context.Context, r *pb.DealRequest) (*pb.Empty, er
 	return &pb.Empty{}, nil
 }
 
+func (h *Hub) ApproveDeal(ctx context.Context, r *pb.DealRequest) (*pb.Empty, error) {
```

Comment:
Wrong argument type for this method: deal id is also required, while spec hash is unneeded.

But it's okay, because it is unimplemented now.